docs/conf.py

```diff
@@ -18,13 +18,16 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
+autodoc_mock_imports = ["adafruit_featherwing", "shared"]
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3.4', None),
     'BusDevice': ('https://circuitpython.readthedocs.io/projects/bus_device/en/latest/', None),
     'Register': ('https://circuitpython.readthedocs.io/projects/register/en/latest/', None),
     'CircuitPython': ('https://circuitpython.readthedocs.io/en/latest/', None),
-    'motor': ('https://circuitpython.readthedocs.io/projects/motor/en/latest/', None)}
+    'motor': ('https://circuitpython.readthedocs.io/projects/motor/en/latest/', None),
+    'seesae': ('https://circuitpython.readthedocs.io/projects/seesaw/en/latest/', None),
```

Comment:
typo...'seesaw`: (don't think this is related to Travis failure)

Reply:
I noticed this and had already fixed it, hadn't pushed yet.